src/styles/starts.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+  justify-content: flex-end;
+
+  &__star {
+    margin-right: 4px;
+    height: 16px;
+    width: 16px;
+    background-image: url('../images/star-active.svg');
+    background-repeat: no-repeat;
+    background-position: center;
```

Comment:
It can be done shorter. [Background](https://developer.mozilla.org/en-US/docs/Web/CSS/background)